src/components/Home.js

```diff
@@ -0,0 +1,84 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { Link } from 'react-router-dom';
+import { categories } from './data';
+import {
+  getForex, getCryptos, getStocks, getEFTs,
+} from '../redux/actions/markets';
+import '../styles/Home.css';
+
+const Home = () => {
```

Comment:
According to the project requirements:

![image](https://user-images.githubusercontent.com/57421823/157003157-9a9afcd2-9554-41e0-a08c-5d441b40eb6a.png)


- [ ] Please, kindly implement a filter for your main categories (Forex, Majors, Cryptos, Stocks, ETF iShares... etc).  

## Explanation

This filter should be some parameter, like a number (e.g. capitalization), letters (e.g. search box for the categories names), or a drop-down list.  The main goal is that only the categories that apply appear when filtered.


## Hints

You could create a component for the filtered rendering, and add it to your main page.  Then, connect it to the Redux store to manage the state of the filtering.